frontend/test/pytest/test_measurement_transforms.py

```diff
@@ -196,6 +198,7 @@ def basic_circuit(theta: float):
         assert np.isclose(eigval_counts_res[-1], counts_expected[-1], atol=100)
         assert np.isclose(eigval_counts_res[1], counts_expected[1], atol=100)
 
+    @flaky
```

Comment:
Why do those tests become flaky?

Reply:
They didn't become flaky here, I just realized running this test file while working on the PR that they fail often enough to be irritating.

I may also have overdone it, I put it on most of the tests that are probabilistic, because I kept having tests fail with just one test that was just a bit off (different one each time). I could also increase `atol` or number of shots if that's preferable.

Reply:
1) Can you make sure we cannot seed the test?

2) If it cannot be seeded, can you specify the flaky parameters to make it a bit more clear? e.g. Also if you can double check which one are to be marked with flaky.

```
@flaky(max_runs=3, min_passes=2)
```

Reply:
I couldn't figure out a way to seed the test,  but some small adjustments to number of shots and `atol`, I was able to set the test file to run 30 times in a row with no failures, so I'll get rid of the `flaky` tag :)

Reply:
Ok thanks :+1: